cabal.project.8.4.4

```diff
@@ -0,0 +1 @@
+cabal.project
```

Comment:
Shouldn't that include the `with-compiler = ghc-8.4.4`?

Reply:
Why? That makes it harder to maintain `cabal.project`

Reply:
well as it is you could do a `cabal  --project-file cabal.project.8.4.4` and it would use the ghc on $PATH (ghc-8.6.5 f. e.) although you has ghc-8.4.4 in $PATH. Maybe doing that makes no sense but then this one could be removed, or is it needed for some reason?

Reply:
> well as it is you could do a `cabal --project-file cabal.project.8.4.4` and it would use the ghc on $PATH (ghc-8.6.5 f. e.)

`cabal --project-file cabal.project.8.4.4` will use `cabal.project.8.4.4.freeze` unconditionally.

Reply:
Yeah, i edited my comment